includes/admin/settings/views/html-admin-page-shipping-zone-methods.php

```diff
@@ -10,6 +10,8 @@
 	<?php echo wc_help_tip( __( 'The following shipping methods apply to customers with shipping addresses within this zone.', 'woocommerce' ) ); ?>
 </h2>
 
+<?php do_action('wc_shipping_zone_before_methods_list'); ?>
```

Comment:
Ok, but change to `do_action( 'woocommerce_shipping_zone_before_methods_table' )`